platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -236,9 +236,8 @@ The `website` and `vcsUrl` properties of [`gradlePlugin`](javadoc/org/gradle/plu
 
 ## Fixed issues
 
-<!--
-This section will be populated automatically
--->
+[#30272](https://github.com/gradle/gradle/issues/30272) Broken equals() contract for LifecycleAwareProject
```

Comment:
❌ That section is populated automatically, with all issues fixed in 8.10.*

These need to go at the top of the file